### DISCUSSION TITLE
Could you add Scale Factor for 4 screens ?

### DISCUSSION THREAD

**Discussion by Paullux** [stoatchat/discussions#106] [Feature Suggestions]
In discord i can change the scale factor, how to do it in Revolt ?

**Comment by sussycatgirl** [comment]
Have you tried using your browser's zoom feature? Crtl and + or - should work in the electron app too.

**Comment by thantik** [comment]
@janderedev this capability should be more discoverable.  I had to hunt this down in order to find out how to scale.  On discord, it's a slider.
